src/sagemaker/fw_utils.py

```diff
@@ -49,8 +49,8 @@
     "Please set the argument \"py_version='py3'\" to use the Python 3 {framework} image."
 )
 PARAMETER_SERVER_MULTI_GPU_WARNING = (
-    "You have selected a multi-GPU training instance type. "
-    "You have also enabled parameter server for distributed training. "
+    "If you have selected a multi-GPU training instance type, "
+    "and have also enabled parameter server for distributed training. "
     "Distributed training with the default parameter server configuration will not "
```

Comment:
Should this have a comma after the "If" portion?
```suggestion
    "and have also enabled parameter server for distributed training, "
    "distributed training with the default parameter server configuration will not "
```

Reply:
I want to keep this a pure merge commit, so fixed it in https://github.com/aws/sagemaker-python-sdk/pull/1637